### PR TITLE
added a use multiplier to battle items

### PR DESCRIPTION
**Pull request by LuqDragon**
I thought that the other PR would implement that, but nothing changed, so I resolved to add.

I think this button fits the model, but not discard the idea of a dropdown to choose the multiplier.

I was afraid to change these functions thinking they were used by all the items, but are used just to battle items, so I will try to make more generic as possible in the next updates.

The 'all' option is a little controversial too.
mult2

closes #771

### CODE REVIEW

File: src/components/battleItemContainer.html [hunk 1]
```diff
@@ -2,6 +2,17 @@
     <div class="card-header p-0" data-toggle="collapse" href="#battleItemContainerBody">
       <span>Battle Items</span>
     </div>
+    <button type="button" class="btn btn-sm btn-primary" 
+            style="position: absolute; right: 0px; top: 0px; width: auto; height: 41px;" 
+            data-bind="text: ItemHandler.multipliers[ItemHandler.multIndex()]"
+            onclick="ItemHandler.incrementMultiplier()"
+            oncontextmenu="ItemHandler.decrementMultiplier(); return false"
+            title="Controls"
+            data-toggle="popover"
```
Comment (RedSparr0w) [inline src/components/battleItemContainer.html:11]: I think we should use a `tooltip` here to match the style of everything else.

Comment (LuqDragon) [reply src/components/battleItemContainer.html:11]: Ah, ok.

First, I thought that, but I found this popover and decided to try it.

**Review by LuqDragon — COMMENTED** [review]
nice update, this simplifies so much xD

File: src/components/battleItemContainer.html (hunk 1 above)
Comment (RedSparr0w) [reply src/components/battleItemContainer.html:11]: Actually, maybe we should leave it, and move the other things over to popovers instead.
It does look nicer IMO.

Comment (LuqDragon) [reply src/components/battleItemContainer.html:11]: maybe I just change the title to 'Use Multiplier' instead 'Controls', the problem is that this popover is relatively big than a tooltip

**Review by RedSparr0w — APPROVED** [review]
👍 Thanks for the PR!

**Pull request merged**